config/crd/kustomization.yaml

```diff
@@ -9,14 +9,6 @@ resources:
 # +kubebuilder:scaffold:crdkustomizeresource
 
 patchesStrategicMerge:
-# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
-# patches here are for enabling the conversion webhook for each CRD
-- patches/webhook_in_functionmeshes.yaml
-- patches/webhook_in_functions.yaml
-- patches/webhook_in_sources.yaml
-- patches/webhook_in_sinks.yaml
-# +kubebuilder:scaffold:crdkustomizewebhookpatch
-
```

Comment:
I recommended keeping this content, it will affect the CRDs generated by `kustomize`.